models/junos/l3_interfaces/junos_l3_interfaces.yml

```diff
@@ -0,0 +1,76 @@
+---
+GENERATOR_VERSION: '1.0'
+ANSIBLE_METADATA: |
+  {
+     'metadata_version': '1.1',
+     'status': ['preview'],
+     'supported_by': 'network'
+  }
+
+NETWORK_OS: junos
+RESOURCE: l3_interfaces
+COPYRIGHT: Copyright 2019 Red Hat
+LICENSE: gpl-3.0.txt
+
+DOCUMENTATION: |
+  module: junos_l3_interfaces
+  version_added: 2.9
+  short_description: Manage Layer-3 interface on Juniper JUNOS devices
+  description: This module provides declarative management of a Layer-3 interface on Juniper JUNOS devices
+  author: Daniel Mellado (@dmellado)
+  notes:
+    - 'Tested against JunOS v18.4R1'
+  options:
+    config:
+      description: A dictionary of Layer-3 interface options
+      type: list
+      elements: dict
+      suboptions:
+        name:
+          description:
+            - Full name of interface, e.g. ge-0/0/1
+          type: str
+          required: True
+        unit:
+          description:
+            - Logical interface number. Value of C(unit) should be of type integer
+          default: 0
+        ipv4:
+          description:
+            - IPv4 addresses to be set for the Layer 3 logical interface mentioned in I(name) option.
+              The address format is <ipv4 address>/<mask>, the mask is number in range 0-32
+              for example, 192.0.2.1/24, or C(dhcp) to query DHCP for an IP address
+          type: list
+          elements: dict
+          suboptions:
+            address:
+              description:
+                - IPv4 address to be set for the specific interface
+              type: str
+        ipv6:
+          description:
+            - IPv6 addresses to be set for the Layer 3 logical interface mentioned in I(name) option.
+              The address format is <ipv6 address>/<mask>, the mask is number in range 0-128
+              for example, 2001:db8:2201:1::1/64 or C(auto-config) to use SLAAC
```

Comment:
same comment about moving the detailed description to the suboption below.

Reply:
ditto